aspnetcore/security/app-secrets.md

```diff
@@ -223,6 +223,30 @@ User secrets can be retrieved via the `Configuration` API:
 
 ::: moniker-end
 
+## Map secrets to a POCO
+
+Mapping an entire object literal to a POCO (a simple .NET class with properties) is useful for aggregating related properties. Consider the following `Movies` object literal:
+
+[!INCLUDE[secrets.json file](~/includes/app-secrets/secrets-json-file.md)]
+
+To map the preceding object literal to a POCO (a simple .NET class with properties), use the `Configuration` API's [object graph binding](xref:fundamentals/configuration/index#bind-to-an-object-graph) feature. The following code binds to a custom `MovieSettings` POCO and accesses the `ServiceApiKey` property value:
```

Comment:
Probably don't need to repeat the parenthetical info: "(a simple .NET class with properties)"

*ooooh* **_feature_** 🙈 ... the banned word! :smile: lol

Reply:
Nice catch. Copy & paste error.